extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/Remote.java

```diff
@@ -24,8 +24,25 @@
 @Qualifier
 public @interface Remote {
     /**
-     * The remote cache name. If no value is provided the default cache is assumed.
```

Comment:
@wburns I did this change since there is no default cache anymore, what do you think about this change?

Reply:
There is a default cache, it is just optional now. I guess I can be convinced either way. One thing though is this will cause a compilation for upgrades, but that should be pretty simple to fix.

Reply:
I will leave as it is now then